wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,207 @@
 | :-- | :-- |
-| `Ctrl` + `B` | Add bookmark at current location. |
-| `Ctrl` + `Shift` + `B` | Remove bookmark at current location. |
-| `Ctrl` + `Right` Arrow | Next bookmark. |
-| `Ctrl` + `Left` Arrow | Previous bookmark. |
-
-At **bottom-right**, you can adjust the **Playback Rate** by **per quarter value.**
+| Blue | Selected note(s) |
+| White/Pink/Yellow | Unselected note colours |
 
 ### Sampling
 
-![Sample pop-up window (Windows 8's ver.)](img/Sample_omc.jpg "Sample pop-up window (Windows 8's ver.)")
-
-**Sampling** is a process of **adding sound to the note.** To begin, select the note and **`Alt` + Left Click** on the selected note (a pop-up window will be shown with sound samples inside your beatmap folder).
+**Sampling** is the process of adding sound to the note. To begin, select the note and `Alt` **+** `Left click` on the selected note and a pop-up window will be shown with sound samples inside the beatmap folder.
 
 #### Sample import
 
-**Left-section** is the **list of sound files you have in the beatmap's folder.**
+![](img/sample-import.jpg "Sample import window")
+
+**Left-section** is the list of sound files there are in the beatmap's folder.
 
-**Right-section** is **settings for the selected sound file.**
+**Right-section** are settings for the selected sound file.
 
-##### Basics
+##### Basic
 
 | Name | Description |
 | :-- | :-- |
-| CustomSet | **Ignoring** selected custom hitsound in this editor, and apply the **default** hitsound **SET** and volume without using timing point. ( Use basic sample must be enable in order to use this to be able to work as its intended )ex : added soft-hitwhistle77.wav with 90% volume.Note that you must add the whistle to the note manually as well. **Also this feature is not recommended while the normal APPLY function in this editor is easier to be used than this one while both of this basically works the same.** |
-| Volume | How loud you want this sound file be. Use integer value only. ( 8 - 100 ) |
-| Use basic sample | **Ignoring** selected custom hitsound in this editor , and apply only the volume on selected notes.This means , no matter what the notes is about, this feature only changes the selected notes volume.Ex : this can be used to change multiple volume on different notes with different hitsound sample at once or just enhance the default hitsound ignoring any timing point's volume setting. |
+| `CustomSet` | **Ignores** selected custom hitsound in this editor, and applies the **default** hitsound set and volume without using timing point. *Notice: `Use basic sample` must be enabled in order to use this to be able to work as it's intended.* |
+| `Volume` | How loud this sound file should be. Use only integer values from 8 to 100. |
+| `Use basic sample` | **Ignores** selected custom hitsound in this editor, and applies only the volume on selected notes. This means, no matter what the notes are about, this feature only changes the selected notes volume. |
```

Comment:
should roughly follow the rewrite I suggested for `CustomSet`

Reply:
I think so -- I tried and IMO the effect is the same as applying sampleset/addition overrides from the left panel, but with the benefit of customizable volume